homework2/Program.cs

```diff
@@ -8,8 +8,49 @@ namespace homework2
 {
     class Program
     {
+        enum TransactionType
```

Comment:
It is better to have new types in a separate file. Please move enum from Program.cs to TransactionType.cs. You can add a new file by right clicking on the project name and then selecting "add" > "class..."